product_docs/docs/migration_toolkit/55/07_invoking_mtk/08_mtk_command_options.mdx

```diff
@@ -450,13 +450,13 @@ The right side specifies a condition that must be true for each row migrated.
 
 For example, this code migrates only those countries with a `COUNTRY_ID` value that isn't equal to `AR`: 
 
-### `COUNTRIES=COUNTRY_ID<>'AR'`
+#### `COUNTRIES=COUNTRY_ID<>'AR'`
```

Comment:
This appears to have been changed from an h3 to an h4 heading, but I'm not sure why it is a heading at all instead of just text.

Instead of:
> #### `COUNTRIES=COUNTRY_ID<>'AR'`

why not?
> `COUNTRIES=COUNTRY_ID<>'AR'`